lib/core/test/unit/Network/Wai/Middleware/LoggingSpec.hs

```diff
@@ -192,6 +192,7 @@ spec = describe "Logging Middleware"
                     , replicate (n - i) (get ctx "/get")
                     ]
             void $ mapConcurrently id reqs
+            threadDelay 1000000 -- let iohk-monitoring flush the logs
```

Comment:
Hmmm. This is a `threadDelay` of 1s inside a property.  This will take more than a minute to run :/ 
Can't we force-stop the monitoring switchboard here to effectively flush the logs? Or, at the very least, only add this on for Windows tests?

Reply:
I was looking for a way to get iohk-monitoring to flush logs but didn't find anything yet.
Unlike the other tests, in this case, the threadDelay is not even effective, and this test still fails.

Reply:
I see. I'd suggest to not spend too much time on this and a possible remedy for this one might be to disable it on windows. This is testing precise timing and interleaving of log requests.

Reply:
Yes good idea. I have made this particular spec pending (only on windows), and changed the other `threadDelay` so that it is only used on Windows.